tools/scripts/deploy.sh

```diff
@@ -0,0 +1,117 @@
+            --target-pass)        TARGET_PASS="$2"; shift; shift ;;
+            -- ) shift; break ;;
+            * ) err "unsupported argument $1"; usage; exit 1 ;;
+        esac
+    done
+
+    echo "Start"
+    tmpdir=$(mktemp -d)
+    mkdir -p ${tmpdir}/opt/prplmesh
+    mkdir -p ${tmpdir}/usr
+
+    cp -r ${BUILD_DIR}/out/* ${tmpdir}/opt/prplmesh
+    mv ${tmpdir}/opt/prplmesh/lib ${tmpdir}/usr
+
+    dbg "create prplmesh.tar.gz..."
+    tar czvf ${BUILD_DIR}/prplmesh.tar.gz -C ${tmpdir}/ .
+    rm -rf ${tmpdir}
+
+    [ "$DEPLOY" = "true" ] && {
+        
+        if [ -n "$PROXY_IP" -a -n "$PROXY_USER" -a -n "$PROXY_PASS" ]; then
+            echo "local md5sum"
+            run md5sum ${BUILD_DIR}/prplmesh.tar.gz
+            
+            echo "scp ${BUILD_DIR}/prplmesh.tar.gz to ${PROXY_USER}@${PROXY_IP}"
+            run sshpass -p ${PROXY_PASS} scp ${SSH_OPTIONS} ${BUILD_DIR}/prplmesh.tar.gz ${PROXY_USER}@${PROXY_IP}:/home/${PROXY_USER}/
+
+            echo "scp ${PROXY_USER}@${PROXY_IP}:/home/${PROXY_USER} to ${TARGET_USER}@${TARGET_IP}:/tmp"
+            run sshpass -p ${PROXY_PASS} ssh ${SSH_OPTIONS} ${PROXY_USER}@${PROXY_IP} "sshpass -p ${TARGET_PASS} scp ${SSH_OPTIONS} /home/${PROXY_USER}/prplmesh.tar.gz ${TARGET_USER}@${TARGET_IP}:/tmp/"
+
+            echo "remote md5sum"
+            run sshpass -p ${PROXY_PASS} ssh ${SSH_OPTIONS} ${PROXY_USER}@${PROXY_IP} "sshpass -p ${TARGET_PASS} ssh ${SSH_OPTIONS} ${TARGET_USER}@${TARGET_IP} md5sum /tmp/prplmesh.tar.gz"
+
+            echo "backaup /opt/prplmesh/beerocks_agent.conf on target"
+            run sshpass -p ${PROXY_PASS} ssh ${SSH_OPTIONS} ${PROXY_USER}@${PROXY_IP} "sshpass -p ${TARGET_PASS} ssh ${SSH_OPTIONS} ${TARGET_USER}@${TARGET_IP} cp /opt/prplmesh/config/beerocks_agent.conf /tmp/"
+
+            echo "untar /tmp/prplmesh.tar.gz on target"
+            run sshpass -p ${PROXY_PASS} ssh ${SSH_OPTIONS} ${PROXY_USER}@${PROXY_IP} "sshpass -p ${TARGET_PASS} ssh ${SSH_OPTIONS} ${TARGET_USER}@${TARGET_IP} tar -C / -xzvf /tmp/prplmesh.tar.gz"
+
+            echo "restore /opt/prplmesh/beerocks_agent.conf on target"
+            run sshpass -p ${PROXY_PASS} ssh ${SSH_OPTIONS} ${PROXY_USER}@${PROXY_IP} "sshpass -p ${TARGET_PASS} ssh ${SSH_OPTIONS} ${TARGET_USER}@${TARGET_IP} cp /tmp/beerocks_agent.conf /opt/prplmesh/config/"
+    
+        # else
+            # TODO COMPLETE BY CORAL
+
+            # sshpass -p ${TARGET_PASS} scp -oUserKnownHostsFile=/dev/null -oStrictHostKeyChecking=no ${BUILD_DIR}/prplmesh.tar.gz ${TARGET_USER}@${TARGET_IP}:/tmp/
+
+            # echo "untar prplmesh in target"
+            # ssh -oUserKnownHostsFile=/dev/null -oStrictHostKeyChecking=no root@192.168.1.1 'tar -xzvf /tmp/prplmesh.tar.gz -C /'
+
+            # echo "Deploy prplmesh completed, on target run /etc/init.d/prplmesh.sh restart for changes to take effect."
+        fi
+    }
+
+    success "Done! Run /etc/init.d/prplmesh.sh restart on target to apply new settings."
```

Comment:
Change to `/opt/prplmesh/scripts/prplmesh_utils.sh restart` so it will work for rdkb also

Reply:
This is marked as resolved but the change hasn't been done...